libs/utils/platforms/juno_energy.py

```diff
@@ -0,0 +1,109 @@
+from energy_model import ActiveState, EnergyModelNode, PowerDomain, EnergyModel
+
+from collections import OrderedDict
+
+a53_cluster_active_states = OrderedDict([
+    (450000, ActiveState(power=26)),
```

Comment:
It can be added (and the code will work), but I think it would be more confusing. The fact that we report capacity at cluster-level in the EAS product codeline's DT is a *bad thing* IMO (exposure of an internal implementation convenience) and definitely won't be reflected in the mainline. So I think we should not add it here.